README.md

```diff
@@ -95,6 +102,14 @@ $ docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/zip_codes.sql
 $ docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/census_tracts.sql
 ```
 
+>For Windows OS please use the following:
+```bash
```

Comment:
These instructions don't quite work on my Win 10 machine in the Command Prompt.

They should probably be:
```cmd
> docker-compose exec -T mysql mysql -u suyc -psuyc suyc
mysql> TRUNCATE census_boundaries;
mysql>TRUNCATE zip_boundaries;
mysql> exit
> docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/zip_codes.sql
> docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/census_tracts.sql
```
Make sure to notice that I changed `bash` to `cmd`.

Reply:
Also, please move the whole `Updating boundaries` section, including your changes, to above the `Importing Census and Zip Code boundaries` section.